Nos/Models/Event+CoreDataClass.swift

```diff
@@ -725,12 +725,10 @@ public class Event: NosManagedObject {
         }
         
         // Did we unfollow someone? If so, remove them from core data
-        if originalFollows.count > newFollows.count {
-            let removedFollows = Set(originalFollows.values).subtracting(newFollows)
-            if !removedFollows.isEmpty {
-                print("Removing \(removedFollows.count) follows")
-                Follow.deleteFollows(in: removedFollows, context: context)
-            }
+        let removedFollows = Set(originalFollows.values).subtracting(newFollows)
```

Comment:
I believe this operation should have a runtime performance of `O(n)`, which in the grand scheme of things, doesn't make this function that much more expensive.